test_scripts/API/VehicleData/StabilityControlsStatus/020_UnsubscribeVehicleData_StabilityControlsStatus_partially_disallowed_by_policy.lua

```diff
@@ -0,0 +1,44 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0253-New-vehicle-data-StabilityControlsStatus.md
+--
+-- Description: UnsubscribeVehicleData RPC with `stabilityControlsStatus` parameter 
+-- which is NOT allowed by Policies and other parameters which is allowed by Policies
```

Comment:
@LitvinenkoIra parameters which are allowed

Reply:
@AByzhynar Done in 4689ec2